File: src/main/java/com/wafflestudio/draft/dto/response/RegionResponse.java
package com.wafflestudio.draft.dto.response;

import com.wafflestudio.draft.model.Region;
import lombok.Data;

@Data
public class RegionResponse {
    private Long id;
    private String name;
    private String depth1;
    private String depth2;
    private String depth3;

    public RegionResponse(Region region) {
        this.id = region.getId();
        this.name = region.getName();
        this.depth1 = region.getDepth1();
        this.depth2 = region.getDepth2();
        this.depth3 = region.getDepth3();
    }
}

File: src/main/kotlin/com/wafflestudio/draft/dto/response/RegionResponse.kt
package com.wafflestudio.draft.dto.response

import com.wafflestudio.draft.model.Region

data class RegionResponse(
        var id: Long? = null,
        var name: String? = null,
        var depth1: String? = null,
        var depth2: String? = null,
        var depth3: String? = null
) {
    constructor(region: Region) : this(
            region.id,
            region.name,
            region.depth1,
            region.depth2,
            region.depth3
    )
}
